lang/lang.nl.json

```diff
@@ -0,0 +1,929 @@
+          "",
+          "{{SETTING_LIST}}",
+          "\u2800"
+        ],
+        "fields": [
+          {
+            "name": "{{REF:fields.actions}}",
+            "value": [
+              "`{{REF:bot.prefix}} {{REF:commands.me}}` - Beheer je gebruikersinstellingen."
+            ]
+          }
+        ],
+        "color": "{{REF:colors.success}}"
+      },
+      "setCompletedBot": {
+        "description": [
+          "{{REF:partials.congratulations}}",
+          "",
+          "De bot setup is voltooid",
+          "",
+          "{{SETTING_LIST}}",
+          "\u2800"
+        ],
+        "fields": [
+          {
+            "name": "{{REF:fields.actions}}",
+            "value": [
+              "`{{REF:bot.prefix}} {{REF:commands.bot}} <{{REF:arguments.bot}}>` - Beheer botinstellingen."
+            ]
+          }
+        ],
+        "color": "{{REF:colors.success}}"
+      },
+      "setDeniedUser": {
+        "description": "Gebruiker setup geweigerd. Je instellingen zijn niet veranderd.",
+        "color": "{{REF:colors.error}}"
+      },
+      "updatedSettingGuild": {
+        "description": "De server's**{{SETTING_NAME}}** instelling is veranderd naar **{{SETTING_VALUE}}**.",
+        "color": "{{REF:colors.success}}"
+      },
+      "updatedSettingBot": {
+        "description": "{{BOT}}'s **{{SETTING_NAME}}** instelling is veranderd naar **{{SETTING_VALUE}}**.",
+        "color": "{{REF:colors.success}}"
+      },
+      "updatedSettingUser": {
+        "description": "Jouw **{{SETTING_NAME}}** instelling is veranderd naar **{{SETTING_VALUE}}**.",
+        "color": "{{REF:colors.success}}"
+      },
+      "removedSettingGuild": {
+        "description": "De server's **{{SETTING_NAME}}** instelling is verwijderd.",
+        "color": "{{REF:colors.success}}"
+      },
+      "removedSettingBot": {
+        "description": "{{BOT}}'s **{{SETTING_NAME}}** instelling is verwijderd.",
+        "color": "{{REF:colors.success}}"
+      },
+      "removedSettingUser": {
+        "description": "Jouw **{{SETTING_NAME}}** instelling is verwijderd",
+        "color": "{{REF:colors.success}}"
+      },
+      "removedGuild": {
+        "description": "De gegevens van deze server zijn verwijderd.",
+        "color": "{{REF:colors.success}}"
+      },
+      "removedBot": {
+        "description": "De gegevens van {{BOT}} zijn verwijderd.",
+        "color": "{{REF:colors.success}}"
+      },
+      "removedUser": {
+        "description": "Jouw gegevens zijn verwijderd.",
+        "color": "{{REF:colors.success}}"
+      },
+      "addedListItem": {
+        "description": "**{{TIME_ZONE}}** is toegevoegd aan de lijst met tijdzones van de server.",
+        "color": "{{REF:colors.success}}"
+      },
+      "removedListItem": {
+        "description": "**{{TIME_ZONE}}** is verwijdered van de lijst met tijdzones van de server.",
+        "color": "{{REF:colors.success}}"
+      }
+    },
+    "validation": {
```

Comment:
I've added one new item here since this translation (it is the first under the validation section), it looks like so in English:
```json
"devOnlyCommand": {
    "description": "This command can only be used by developers.",
    "color": "{{REF:colors.warning}}"
},
```